packages/data-service/src/connect-mongo-client.spec.ts

```diff
@@ -252,8 +252,10 @@ describe('connectMongoClient', function () {
         expect(error).to.be.instanceOf(Error);
 
         // propagates the tunnel error
-        expect(error.message).to.match(
-          /(All configured authentication methods failed|ENOTFOUND compass-tests\.fakehost\.localhost)/
+        // NOTE: this heavily depends on which server version we're running on
```

Comment:
do we have to test for a specific message? can't we just confirm an error?

Reply:
I don't know. I didn't write the original test and I don't want to unnecessarily change the intention.

Reply:
or, if we want to test the propagation, is there a way to mock the error?

Reply:
Yes, but this file is an integration test, not a unit test.